app/main/views/login.py

```diff
@@ -1,25 +1,16 @@
 from __future__ import absolute_import
-import six
 from flask_login import current_user
-from flask import current_app, flash, redirect, render_template, url_for, abort
+from flask import flash, redirect, render_template, url_for, current_app
 
 from dmapiclient.audit import AuditTypes
-from dmutils.email import generate_token, send_email
-from dmutils.email.exceptions import EmailError
+from dmutils.email import send_user_account_email
 
 from .. import main
 from ..forms.auth_forms import EmailAddressForm
-from ..helpers import hash_email, login_required
```

Comment:
Is `hash_email` used by anything else or could we delete the helper function as well? There's a shared one in dmutils now that we should probably switch to if it is used.

Reply:
It's used in app/main/view/frameworks, but could probable update to use the utils one. I'm going to check they have the same functionality though.

Reply:
Yep, have removed the helper and used the one from utils.